### PR TITLE
added calculator.py

### PR DESCRIPTION
**Pull request by shreydan**
Uses Python's eval() function to implement a calculator.

Sample output:

```
Scientific Calculator
Eg: pi * sin(90) - sqrt(81)

What is (1/3) * pi * 8 * 8 * 12

804.247719318987
```

More programs at: [shreydan/Python](https://github.com/shreydan/Python)

### CODE REVIEW

File: calculator.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
+"""
+Written by: Shreyas Daniel - github.com/shreydan
+Description: Uses Pythons infamous eval() function as a way to implement calculator
```
Comment (ankit167) [inline calculator.py]: line too long, exceeds 80 characters.

File: calculator.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+"""
+
+import math
+
+
+def calc(k):
+
+    functions = ['sin','cos','tan','sqrt','pi']
```
Comment (ankit167) [inline calculator.py]: Pls use spaces after comma.

File: calculator.py [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+Description: Uses Pythons infamous eval() function as a way to implement calculator
+"""
+
+import math
+
+
+def calc(k):
+
+    functions = ['sin','cos','tan','sqrt','pi']    
+
```
Comment (ankit167) [inline calculator.py]: Extra line is not required.

File: calculator.py [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+        print ("Can't divide by 0")
+        exit()
+    except NameError:
+        print ("Invalid input")
+        exit()
+    
+    return k
+
+
+print ("\nScientific Calculator\nEg: pi * sin(90) - sqrt(81)")
```
Comment (ankit167) [inline calculator.py]: Pls wrap function call inside main() function.

**Comment by shreydan** [comment]
Done as told:

- removed unnecessary lines
- wrapped inside main()
- added space after commas
 

Also added % option and more comments.

**Comment by geekcomputers** [comment]
Thank you for you contribution to the repo and the community

**Pull request merged**